lib/ValidateSearchApiVersion.js

```diff
@@ -1,6 +1,6 @@
 const supportedVersions = {
-  alternatives: ['2017-11-11-Preview'],
-  default: '2017-11-11',
+  alternatives: ['2017-11-11', '2017-11-11-Preview'],
```

Comment:
I'm wary about keeping support for the preview versions we will have to remove it eventually so why not now but this is safe in that no consumers will break with this change.